### DISCUSSION TITLE
Main Window Background Transparent

### DISCUSSION THREAD

**Discussion by claudioluciano** [emilk/egui#245] [Q&A] [answered]
Heyy o/
It's possible to make the background the main window transparent? **_(not the widgets window)_**
I'm trying to use with the **winit** transparent feature.

**Comment by emilk** [comment, marked as answer]
Hi! I thought this was a fun idea, so I added it as a feature in https://github.com/emilk/egui/commit/3450168e94898317cddfca512ebc478b244403ac

Enjoy :)

**Reply by claudioluciano** [reply]
Wow thanks : )
